File: python_programming_basics/conditional_statements_advanced/e_05_journey.py
budget = float(input())
season = input()
destination = ""
place = ""

if budget <= 100:
    destination = "Bulgaria"

    if season == "summer":
        place = "Camp"
        budget *= 0.3

    else:
        place = "Hotel"
        budget *= 0.7

elif budget <= 1000:
    destination = "Balkans"

    if season == "summer":
        place = "Camp"
        budget *= 0.4

    else:
        place = "Hotel"
        budget *= 0.8

else:
    destination = "Europe"
    place = "Hotel"
    budget *= 0.9

print(f"Somewhere in {destination}")
print(f"{place} - {budget:.2f}")

File: JavaScript Basics/03. Conditional Statements Advanced/e-05-Journey.js
function vacationPlan(budget, season) {
    let destination = "";
    let vacationType = "";
    let spentMoney = 0.0;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (season === "summer") {
            vacationType = "Camp";
            spentMoney = budget * 0.30;
        } else if (season === "winter") {
            vacationType = "Hotel";
            spentMoney = budget * 0.70;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        if (season === "summer") {
            vacationType = "Camp";
            spentMoney = budget * 0.40;
        } else if (season === "winter") {
            vacationType = "Hotel";
            spentMoney = budget * 0.80;
        }
    } else {
        destination = "Europe";
        vacationType = "Hotel";
        spentMoney = budget * 0.90;
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${vacationType} - ${spentMoney.toFixed(2)}`);
}


vacationPlan(50, "summer");
vacationPlan(75, "winter");
vacationPlan(312, "summer");
vacationPlan(678.53, "winter");
vacationPlan(1500, "summer");
